install.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+#
+# This is a simple installer script for KubeOne 
+#
+#
+# What OS is used
+OS=$(uname)
+# find out what's the latest version
+VERSION=$(curl -w '%{url_effective}' -I -L -s -S https://github.com/kubermatic/kubeone/releases/latest -o /dev/null | sed -e 's|.*/v||')
+# download URL for the latest version
+URL="https://github.com/kubermatic/kubeone/releases/download/v${VERSION}/kubeone_${VERSION}_${OS}_amd64.zip"
+echo $OS
+echo $VERSION
+echo $URL
+
+# 'kubeone' will be installed into this dir:
+DEST=/usr/local/bin
+
+# Download the latest version for the OS and save it as zip
+curl -LO $URL
+
+if [ $? -eq 0 ]
+then 
+    echo "Copying kubeone binary into $DEST"
+    # unpack:
+    unzip kubeone_${VERSION}_${OS}_amd64.zip -d kubeone_${VERSION}_${OS}amd64
```

Comment:
```suggestion
    unzip kubeone_${VERSION}_${OS}_amd64.zip -d kubeone_${VERSION}_${OS}_amd64
```
Very tiny nit: I think it would look better with `_` between `OS` and `arch`. It also matches the archive name that way.